src/polyswarm_api/__init__.py

```diff
@@ -300,7 +300,7 @@ def _fix_result(self, result):
         while True:
             result = await self.lookup_uuid(uuid)
 
-            if result.get('status', 'error') == 'error':
```

Comment:
This doesn't properly handle exceptions coming out of the lookup. Previous behavior was that, on server error, this would exit instead of continuing to wait for the timeout.

I don't have a strong opinion on this, but on server error it seems weird to keep going.

Are there any other 'status' values returned by the endpoint that we need to handle here?

Reply:
But it does not keep going in case of a server error, it returns the current value right after this point. I just changed the part where it was overwriting the result with `OK`. The commit that changed that line went to the wrong PR, I moved it back here so that this makes more sense.